src/AuthorizedSocketConnection.js

```diff
@@ -30,7 +30,7 @@ type Subscription = {
 
 type AuthorizedSocketOptions<TContext, TCredentials> = {|
   schema: GraphQLSchema,
-  subscriber: Subscriber,
+  subscriber: Subscriber<*>,
```

Comment:
does this actually give us type checking? or should this be some explicit `TSubscriberOptions`?

Reply:
seemed kinda crazy to pass down TSubscriber options from the top for this...i'm not sure you'd even get the strict check you'd want in that case, afaik this was already an `any` from the subscription side yeah?

Reply:
a bit; could it be inferred? e.g. could we just have `TSubscriber`?

Reply:
sounds reasonable to me. let's use `any` instead of `*`, though, because we're not inferring anything useful here, and let's follow up immediately after clearing the downstream work with actually typing these interfaces